<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_Btn_write</class>
 <widget class="QDialog" name="Dlg_Btn_write">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>251</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>407</width>
    <height>251</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>写入文件</string>
  </property>
  <widget class="QRadioButton" name="radio_default">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>使用默认文件名</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_personal">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>使用自定义文件名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Path">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_write">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>写入</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_cancel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_warning">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red</string>
   </property>
   <property name="text">
    <string>*如果同名文件存在，点击“写入”会导致原来的文件内容被覆盖，请三思</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_default">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>D:/data.txt</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
